### DISCUSSION TITLE
YT tutorial for Alephjs

### DISCUSSION THREAD

**Discussion by Krrishdhaneja** [alephjs/aleph.js#199] [General]
Hey contributors👋, hope so you all are well and good, today I was just seeing my Google Feed and it appeared in that and I was just thinking of using react with deno and this would really help me, so is there any YT tutorial on this cuz i Can't find any so anyone would like to make a tutorial of begginers?

**Comment by shadowtime2000** [comment]
A YT tutorial would be nice. I could create one but it probably won't be that good because it would be voiceless for personal reasons and to retain my anonymity. Docs are prob the best way to go rn but I think a YT tutorial is something we should really like into

**Comment by ije** [comment]
i have a plan to create a tutorial page(may including videos), but i'm too busy currently

**Comment by Krrishdhaneja** [comment]
> A YT tutorial would be nice. I could create one but it probably won't be that good because it would be voiceless for personal reasons and to retain my anonymity. Docs are prob the best way to go rn but I think a YT tutorial is something we should really like into

Anonymity? I don't think anything wrong would happen u could create a faceless video though cuz most of the people think its hard to understand through docs, though I can create one and I do have one YT channel but till the date I haven't learnt this, I think u can provide me with scripting of videos so that there would be no delay?

**Comment by Krrishdhaneja** [comment]
Btw who created/started this project(Alephjs)?

**Comment by shinspiegel** [comment]
> i have a plan to create a tutorial page(may including videos), but i'm too busy currently

I could create or improve a tutorial page. Next weekend I'll make a good starting project and show the main features of this project. If that's doesn't bother you guys.

**Comment by ije** [comment]
@shinspiegel thanks, please go ahead!

**Comment by shadowtime2000** [comment]
> Btw who created/started this project(Alephjs)?

made by @ije.
